tests/bluetooth/tester/src/bttester.c

```diff
@@ -0,0 +1,292 @@
+/* bttester.c - Bluetooth Tester */
```

Comment:
I think this file can be pulled out into an upstream subsys/bluetooth/qualification module, along with common test modules, like GAP, GATT, Mesh and MMDL perhaps. This way, it can be included in test applications in end product verification, as we talked about on Monday. Let's get this PR merged first though.

cc @joerchan @carlescufi

Reply:
Yes, I completely agree @trond-snekvik. We should get this upstream, since we already have the equivalent for regular host, and this only extends it with additional features.